### ISSUE TITLE
Model creation in PyTorch/TensorFlow

### ISSUE THREAD

**Issue by darkestfloyd** [NischalBhatewara/PNet#6]
Creation of [ASN](https://arxiv.org/pdf/1704.07535.pdf) using PyTorch. 

Pipeline:
1. Data Processing
2. Model Creation
3. Tuning on dev set
4. Predict on test set
5. Calculate metrics

**Comment by darkestfloyd** [comment]
Created basic dataloader class to load train, dev, and test set.

**Comment by darkestfloyd** [comment]
Issue too broad, opening sub-issues.
